content/projects/gsoc/index.adoc

```diff
@@ -9,6 +9,7 @@ sig: gsoc
 opengraph:
   image: /images/gsoc/opengraph.png
 links:
+  gitter: "https://gitter.im/jenkinsci/gsoc-sig"
```

Comment:
So this chat was not abandoned? I thought so

Reply:
We are keeping the Gitter channel open for Instant Messaging. And there is still activity there. 
But we are decommissioning the mailing list(s) in favour of community.jenkins.io. (aka let it slowly die)